Assets/Scripts/Audio/FMODRadio.cs

```diff
@@ -16,17 +14,16 @@ void Start()
     {
         interactable = Utils.GetRequiredComponent<Interactable>(this);
         interactable.OnInteractEnd += ChangeStation;
-        musicEv = FMODUnity.RuntimeManager.CreateInstance(Music);
-        musicEv.set3DAttributes(FMODUnity.RuntimeUtils.To3DAttributes(transform.position)); 
-        musicEv.start();
+        musicEv = GetComponent<FMODUnity.StudioEventEmitter>();
```

Comment:
Should probably do `Utils.GetRequiredComponent()` like you did with interactable (Alex always got me good with this so now I know loool)

Reply:
Done!